query/encode.go

```diff
@@ -166,6 +173,8 @@ func reflectValue(values url.Values, val reflect.Value) error {
 				del = ','
 			} else if opts.Contains("space") {
 				del = ' '
+			} else if opts.Contains("key") {
```

Comment:
why "key"?  I'm not sure that I have another word that works better, but "key" doesn't quite seem right.  I wonder if "[]" could work?  So you'd have:

``` go
type A struct {
    B []string `url:"b,[]"`
}
```

too weird to use just symbols in a tag option?